networkx/linalg/laplacianmatrix.py

```diff
@@ -46,6 +46,21 @@ def laplacian_matrix(G, nodelist=None, weight="weight"):
     to_numpy_array
     normalized_laplacian_matrix
     laplacian_spectrum
+
+    Examples
+    --------
+    For graphs with multiple connected components, L is permutation-similar
+    to a block diagonal matrix where each block is the respective Laplacian
+    matrix for each component.
+
+    >>> G = nx.graph_atlas(26) #This graph from the Graph Atlas has 2 connected components.
```

Comment:
Similar comment here to the other PR - let's use explicit examples for the input Graph

Reply:
I'll do it